packages/treasury/bundles/install-on-chain.js

```diff
@@ -20,11 +20,12 @@ const DEFAULT_PROTOCOL_FEE = 6n;
  * @param {Store<NameHub, NameAdmin>} param0.nameAdmins
  * @param {ERef<PriceAuthority>} param0.priceAuthority
  * @param {ERef<ZoeService>} param0.zoe
+ * @param {Handle<'feeMintAccess'>} param0.feeMintAccess
  * @param {NatValue} param0.bootstrapPaymentValue
  * @param {NatValue} [param0.poolFee]
  * @param {NatValue} [param0.protocolFee]
  */
-export async function installOnChain({ agoricNames, board, centralName, chainTimerService, nameAdmins, priceAuthority, zoe, bootstrapPaymentValue, poolFee = DEFAULT_POOL_FEE, protocolFee = DEFAULT_PROTOCOL_FEE }) {
+export async function installOnChain({ agoricNames, board, centralName, chainTimerService, nameAdmins, priceAuthority, zoe, feeMintAccess, bootstrapPaymentValue, poolFee = DEFAULT_POOL_FEE, protocolFee = DEFAULT_PROTOCOL_FEE }) {
```

Comment:
How did this long line ever lint?

Reply:
It might not have linting.

Reply:
I looked into this, and it is because this file is within `bundles/` and our eslint settings ignore files within `bundles/`

<img width="369" alt="Screen Shot 2021-08-12 at 6 45 10 PM" src="https://user-images.githubusercontent.com/2441069/129293253-3d6d6eda-0788-4fe2-9df1-36afe0eae7c3.png">

I'll make an issue for this, I think we want to move the file out of "bundles/" and keep the eslint setting.

Reply:
Issue made here: https://github.com/Agoric/agoric-sdk/issues/3672

Reply:
Also linted the file manually!